### PR TITLE
add auto token mapping

### PR DESCRIPTION
**Pull request by jazzyfresh**
part of https://github.com/pulumi/home/issues/2609

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -248,6 +250,10 @@ func Provider() tfbridge.ProviderInfo {
 		},
 	}
 
+	defaults := x.TokensSingleModule("gitlab_", gitLabMod, x.MakeStandardToken(gitLabPkg))
+	err := x.ComputeDefaults(&prov, defaults)
+	contract.AssertNoError(err)
```
Comment (aq17) [inline provider/resources.go]: Apparently we need to use `AssetNoErrorf` now to make the linter happy, haven't seen that before 
```suggestion
	contract.AssertNoErrorf(err)
```

**Comment by aq17** [comment]
It looks like https://github.com/pulumi/pulumi-gitlab/blob/master/sdk/python/pulumi_gitlab/user_gpg_key.py was unchanged but changes to the examples are expected

**Review by aq17 — APPROVED** [review]
LGTM pending tests!

File: provider/resources.go (hunk 1 above)
Comment (jazzyfresh) [reply provider/resources.go]: race condition: i just pushed that fix, waiting for the tests to rerun

**Comment by jazzyfresh** [comment]
> It looks like https://github.com/pulumi/pulumi-gitlab/blob/master/sdk/python/pulumi_gitlab/user_gpg_key.py was unchanged but changes to the examples are expected

It seems odd that only the python language would remove that example.

**Pull request closed without merging**